src/pages/cheoljinJu/Main/Main.js

```diff
@@ -1,8 +1,42 @@
 import React, { Component } from 'react';
+import Nav from '../../../components/Nav/Nav';
+import Aside from './Aside/Aside';
+import Feed from './Feed/Feed';
+import './Main.scss';
 
 class MainCheoljin extends Component {
+  state = {
+    feeds: [],
+  };
+
+  componentDidMount = () => {
+    fetch('http://localhost:3000/data/CommentDataCJ.json', {
```

Comment:
- local 호스트(내 컴퓨터)에서 목업 데이터를 불러오실때, `http://localhost:3000` 부분은 생략해주세요! 생략하면 자동으로 작성되어 들어가는데, 포트 번호 또한 자동으로 들어갑니다. 
- 포트 번호가 바뀌는 일이 생각보다 자주 있습니다. 현재와 같이 직접 작성을 해주시면, 포트 번호가 바뀔 때마다 일일이 수정을 해주어야 해서 비효율적입니다!
```suggestion
    fetch('/data/CommentDataCJ.json', {
```

Reply:
Mock 데이터를 불러올 때 생략할 수 있는 부분은 몰랐었는데 알려주셔서 감사합니다! 
말씀하신 부분 적용해서 수정도 하고 관련 내용도 더 공부해보도록 하겠습니다!